pipelines/plain_pipelines/build-docker-containers.yml

```diff
@@ -0,0 +1,433 @@
+resources:
+- name: paas-docker-cloudfoundry-tools
+  type: git
+  source:
+    uri: https://github.com/alphagov/paas-docker-cloudfoundry-tools.git
+    branch: master
+
+- name: paas-tech-docs
+  type: git
+  source:
+    uri: https://github.com/alphagov/paas-tech-docs.git
+    branch: master
+
+- name: paas-semver-resource
+  type: git
+  source:
+    uri: https://github.com/alphagov/paas-semver-resource.git
+    branch: gds_master
+
+- name: psql
+  type: registry-image
+  source: &build-image-source
```

Comment:
Could we pull these definitions out in to a meta block, [like we do in the create-cloudfoundry pipeline](https://github.com/alphagov/paas-cf/blob/master/concourse/pipelines/create-cloudfoundry.yml#L56)? IMO it's tidier to have every instance of its usage refer to it in the same way, rather than the first usage defining it.

Reply:
Spoke to Mo about it and agreed he's condensed it about as much as it can be